recipe/meta.yaml

```diff
@@ -9,29 +9,33 @@ source:
   # NOTE: pypi doesnt contain a source package for it. See
   # https://github.com/ray-project/tune-sklearn/issues/159
   url: https://github.com/ray-project/tune-sklearn/archive/v{{ version }}.tar.gz
-  sha256: 8a42e3b4ead768a18a83edb137f00df175abd051fb03b8809b7b66de42d18108
+  sha256: f6d531f99a02de4a420cbefb96de64c8db4a014f2c7352f38a6a7a37e2103614
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<38 or py>310 or aarch64 or s390x or ppc64le]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools
+    - wheel
+    - numpy {{ numpy }}
   run:
-    - python >=3.6
-    - ray-tune >=1.0.1
-    - ray-core >=1.0.1
-    - pandas
-    - numpy >=1.16
+    - python
+    - ray-tune >=2.0.0
+    - ray-core >=2.0.0
```

Comment:
```suggestion
```
`ray-tune >=2.0.0` is enough because it already has `ray-core` inside `ray-defaults`

Reply:
And there is a missing `numpy` https://github.com/ray-project/tune-sklearn/blob/v0.4.6/setup.py#L31. As you add `numpy {{ numpy }}` in `host` then in run should be:
`- {{ pin_compatible('numpy') }}`

Reply:
Thanks, Serhii! Added those changes.